src/python/pants/backend/jvm/tasks/coursier/coursier_subsystem.py

```diff
@@ -49,7 +49,7 @@ def register_options(cls, register):
              ],
              help='Additional options to pass to coursier fetch. See `coursier fetch --help`')
     register('--artifact-types', type=list, fingerprint=True,
-             default=['jar', 'bundle', 'test-jar', 'maven-plugin', 'src', 'doc', 'aar'],
+             default=['jar', 'bundle', 'test-jar', 'maven-plugin', 'src', 'doc'],
```

Comment:
This is necessary because otherwise coursier will grab the `jna-4.5.aar` over the `jar`. Currently we don't have any android users, so it's probably okay.